packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -67,69 +67,6 @@ class GenSnapshot {
   }
 }
 
-/// Dart snapshot builder.
```

Comment:
TIL that the `--url_mapping` flags also do nothing in a Dart 2 world.  /cc @a-siva